trino/trino.go

```diff
@@ -78,31 +78,31 @@ import (
 )
 
 func init() {
-	sql.Register("presto", &sqldriver{})
+	sql.Register("trino", &sqldriver{})
 }
 
 var (
 	// DefaultQueryTimeout is the default timeout for queries executed without a context.
 	DefaultQueryTimeout = 60 * time.Second
 
-	// DefaultCancelQueryTimeout is the timeout for the request to cancel queries in presto.
+	// DefaultCancelQueryTimeout is the timeout for the request to cancel queries in trino.
 	DefaultCancelQueryTimeout = 30 * time.Second
 
 	// ErrOperationNotSupported indicates that a database operation is not supported.
-	ErrOperationNotSupported = errors.New("presto: operation not supported")
+	ErrOperationNotSupported = errors.New("trino: operation not supported")
 
 	// ErrQueryCancelled indicates that a query has been cancelled.
-	ErrQueryCancelled = errors.New("presto: query cancelled")
+	ErrQueryCancelled = errors.New("trino: query cancelled")
 )
 
 const (
 	preparedStatementHeader = "X-Presto-Prepared-Statement"
-	preparedStatementName   = "_presto_go"
-	prestoUserHeader        = "X-Presto-User"
-	prestoSourceHeader      = "X-Presto-Source"
-	prestoCatalogHeader     = "X-Presto-Catalog"
-	prestoSchemaHeader      = "X-Presto-Schema"
-	prestoSessionHeader     = "X-Presto-Session"
+	preparedStatementName   = "_trino_go"
+	trinoUserHeader         = "X-Presto-User"
```

Comment:
How about removing the prefix so it's just `userHeader`? Since the package name is already `trino`, the prefix stutters and there's no other `userHeader` in the whole package.

Reply:
It is subjective - but I will keep prefix here. This is constant definition, and having prefix here points out that is constant specific to Trino protocol. Not some generic HTTP header which we happen to use with Trino.